runtime-interface/src/lib.rs

```diff
@@ -72,8 +73,7 @@ impl PassBy for LazyPagesProgramContext {
 #[codec(crate = codec)]
 pub struct LazyPagesRuntimeContext {
     pub page_sizes: Vec<u32>,
-    // TODO: considering change global name types to `TrimmedString` (issue #2098)
```

Comment:
Need to figure out does this changes invalidates runtime interfaces
(check that LimitedStr encodes like String)